app/frontend/src/elm/Component/Account/Page/WaitPayment.elm

```diff
@@ -0,0 +1,197 @@
+module Component.Account.Page.WaitPayment exposing
+    ( Message(..)
+    , Model
+    , initModel
+    , update
+    , view
+    )
+
+import Data.Json
+    exposing
+        ( CreateEosAccountResponse
+        , RailsResponse
+        , createEosAccountResponseDecoder
+        , decodeRailsResponseBodyMsg
+        , railsResponseDecoder
+        )
+import Html
+    exposing
+        ( Html
+        , article
+        , button
+        , div
+        , h2
+        , main_
+        , p
+        , text
+        )
+import Html.Attributes
+    exposing
+        ( attribute
+        , class
+        , type_
+        )
+import Html.Events exposing (onClick, onInput, onSubmit)
+import Http
+import Navigation
+import Translation
+    exposing
+        ( I18n
+            ( AccountCreationFailure
+            , AccountCreationWaitPaymentMsg1
+            , AccountCreationWaitPaymentMsg2
+            , DebugMessage
+            , EmptyMessage
+            , PaymentComplete
+            )
+        , Language
+        , toLocale
+        , translate
+        )
+import Util.Flags exposing (Flags)
+import Util.Urls as Urls
+import View.I18nViews exposing (textViewI18n)
+import View.Notification as Notification
+
+
+
+-- MODEL
+
+
+type alias Model =
+    { orderNo : String
+    , notification : Notification.Model
+    }
+
+
+initModel : Maybe String -> Model
+initModel maybeOrderNo =
+    let
+        orderNo =
+            case maybeOrderNo of
```

Comment:
Maybe.Withdefault 를 사용하면, 한줄로 줄일 수 있습니다.

Reply:
네 수정했습니다~

아래 노트 참고하는 것도 좋을 것 같습니다 :)
https://package.elm-lang.org/packages/elm/core/latest/Maybe#withDefault
Note: This can be overused! Many cases are better handled by a case expression. And if you end up using withDefault a lot, it can be a good sign that a custom type will clean your code up quite a bit!